app/components/file_attachment_list/file_attachment_document.js

```diff
@@ -252,7 +252,7 @@ export default class FileAttachmentDocument extends PureComponent {
         let fileAttachmentComponent;
         if (downloading) {
             fileAttachmentComponent = (
-                <AnimatedCircularProgress
+                <CircularProgress
```

Comment:
Is this just a precaution regarding crash issue in using AnimatedCircularProgress on Android?

Reply:
Nope this is because the AnimatedCircular progress started a new animation anytime the fill value changed, with this it also changes but no need for animation.